client/lib/i18n-utils/switch-locale.js

```diff
@@ -105,6 +107,33 @@ export async function switchCSS( elementId, cssUrl ) {
 	newLink.id = elementId;
 }
 
+function flipRTL( url, isRTL ) {
+	if ( isRTL ) {
+		return url.endsWith( '.rtl.css' ) ? url : url.replace( /\.css$/, '.rtl.css' );
+	}
+
+	return ! url.endsWith( '.rtl.css' ) ? url : url.replace( /\.rtl.css$/, '.css' );
+}
+
+function switchWebpackCSS( isRTL ) {
+	const currentLinks = document.querySelectorAll( 'link[rel="stylesheet"][data-webpack]' );
+
+	return map( currentLinks, async currentLink => {
+		const currentHref = currentLink.getAttribute( 'href' );
+		const newHref = flipRTL( currentHref, isRTL );
+		if ( currentHref === newHref ) {
```

Comment:
Can this ever happen, given what `flipRTL` does?

Reply:
The logic is "put the URL into the desired state (LTR or RTL)" rather than "toggle the state from the current state to the other one". Maybe `flipRTL` has a misleading name?

Reply:
Yeah, might be worth renaming :thinking:

Reply:
Renamed to `setRTLFlagOnCSSLink` and added doc comment to the function.